cinnabar/measurements.py

```diff
@@ -53,19 +54,55 @@ class Measurement(DefaultModel):
 
     @classmethod
     def from_experiment(cls,
-                        pIC50: float,
+                        Ki: unit.Quantity,
                         label: str,
+                        uncertainty: unit.Quantity = 0 * unit.nanomolar,
                         source: str = '',
+                        temperature: unit.Quantity = 298.15 * unit.kelvin,
                         ):
         """Create Measurement from experimental data
 
         Parameters
         ----------
+        Ki: unit.Quantity
+            experimental Ki value
+            ex.: 500 * unit.nanomolar OR 0.5 * unit.micromolar
+        label: str, optional
+            label for this data point.
+        uncertainty: unit.Quantity
+            uncertainty of the experimental value
+            default is zero if no uncertainty is provided (0 * unit.nanomolar)
+        source: str, optional
+            source of experimental measurement
+        temperature: unit.Quantity
+            temperature in K at which the experimental measurement was carried out.
+            By default: 298 K (298.15 * unit.kelvin)
         """
+        R = 1.9872042586408 * 0.001 * unit.kilocalorie_per_mole / unit.kelvin
+        # Convert Ki and uncertainty to molar, then strip unit off for the math
+        Ki = Ki.to(unit.molar).m
+        uncertainty = uncertainty.to(unit.molar).m
+        if Ki > 0:
+            # dG = RT ln Ki
+            DG = R.m * temperature.m * math.log(Ki) * unit.kilocalorie_per_mole
+        else:
+            raise ValueError(
+                "Ki value cannot be zero or negative. "
+                "Check if dG value was provided instead of Ki."
+            )
+        # Convert Ki uncertainty into dG uncertainty: RT * uncertainty/Ki
+        # https://physics.stackexchange.com/questions/95254/the-error-of-the-natural-logarithm
+        if uncertainty >= 0:
+            uncertainty_DG = R.m * temperature.m * uncertainty / \
+                             Ki * unit.kilocalorie_per_mole
```

Comment:
```suggestion
        if uncertainty >= 0 * unit.molar:
            uncertainty_DG = (unit.molar_gas_constantt * temperature.to(unit.kelvin)
                              * uncertainty / Ki).to(unit.kilocalcorie_per_mole)
```
Please double check the output units, but I think that makes sense?

Also just a quick comment on line continuation whilst you're reading the PEP8 spec. Generally implicit line continuations (i.e. using brackets) are prefered over explicit ones (i.e. using `\`). Mostly because `/  \` tends to be really confusing 😅

Reply:
Yes, that looks good! Should I commit the changes or are you doing that?